run_tests.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import os
+import signal
+from subprocess import check_call, Popen, check_output
+import sys
+import time
+import pip
+import urllib2
+import tarfile
+import shutil
```

Comment:
Please sort the imports alphabetically, and move the one where you use `from` to its own block at the end.